website/app/templates/head.hbs

```diff
@@ -0,0 +1,90 @@
+{{!
+
+Per https://github.com/shipshapecode/ember-meta#defining-your-own-headhbs this is a place where we can customize what information is injected by the ember-meta addon
+
+We've made some small customizations to simplify this compared to the default template
+
+}}
+
+{{! Begin General }}
+
+{{! TODO: Determine how to avoid this conflict between ember-meta and ember-page-title where they both set a <title> which causes the latter addon to throw an exception }}
+{{!-- {{#if this.model.title}}
+  <title>{{this.model.title}}</title>
+{{/if}} --}}
```

Comment:
This (lines 12-14) is causing issues in my local linter, are you sure is correct?

Reply:
I'm not. For some reason the linting on the file is behaving quite strangely. Perhaps once I clean it up this will resolve itself.